run-devel.sh

```diff
@@ -26,6 +27,9 @@ while [ -n "$1" ]; do
     ROS_DISTRO=$2
     shift
     ;;
+  -x | --display)
```

Comment:
Why just sharing the `DISLAY` environment variable is not enough?

Reply:
It must have access and permission to the x server socket. Generally, I followed "the simple way" described in [this](http://wiki.ros.org/docker/Tutorials/GUI) tutorial

Reply:
But why specify the `XDISPLAY` vairable? I mean the socket we share at its default locaution `/tmp/.X11-unix` and the display we say which is with `DISPLAY` env. What is the error we get if we do not do this step?

Reply:
You can check by running `gzserver --verbose`. Without the `-x` I get:

`[Err] [RenderEngine.cc:749] Can't open display: :1`